custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
```

Comment:
Consider using template literals for string interpolation to improve readability and maintainability.